src/jitlog/messages.lua

```diff
@@ -1,3 +1,23 @@
+--[[
```

Comment:
I think the long lines in this file could really do with wrapping: reading them in github (and in many text editors) is painful. I tend to wrap to 100 chars, but there is a reasonable argument for 80 chars too.